### PR TITLE
tasks finished

### PR DESCRIPTION
**Pull request by lefteris-L**
[DEMO LINK](https://lefteris-l.github.io/layout_calendar/)

### CODE REVIEW

**Review by SalehAkaJim — APPROVED** [review]
Nice job. Everything seems alright 😀

File: src/index.html [hunk 1]
```diff
@@ -4,9 +4,44 @@
...
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.css">
   </head>
-  <body>
-    <h1>Calendar</h1>
+
+  <body class="page">
+    <div class=
+      "calendar calendar--start-day--wed calendar--month-length--30"
+    >
```
Comment (mgrinko) [inline src/index.html:13]: If you really need to split the line than do it the next way
```suggestion
    <div
      class="calendar calendar--start-day--wed calendar--month-length--30"
    >
```

File: src/styles/utils/_variables.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+$day-h-w: 100px;
+$day-gap: 1px;
+$calendar-padding: 10px;
+$calendar-width:
+  $day-h-w * 7
+  + $day-gap * 6
+  + $calendar-padding * 2;
```
Comment (mgrinko) [inline src/styles/utils/_variables.scss:7]: I'd put it in 1 line :)
but this way is also good

**Review by mgrinko — APPROVED** [review]

File: src/styles/utils/_variables.scss (hunk 2 above)
Comment (lefteris-L) [reply src/styles/utils/_variables.scss:7]: I just find it clearer this way. Thank you @mgrinko

**Pull request closed without merging**